src/app.scss

```diff
@@ -55,7 +55,7 @@ body,
   display: flex;
   justify-content: center;
   padding: 5em;
-  z-index: 10;
+  z-index: 1021;
 
```

Comment:
To avoid this: 

<img width="1109" alt="image" src="https://github.com/os2display/display-admin-client/assets/15377965/1add3db5-6f7e-4951-941f-f6153ddfb65a">